apps/user-profile/src/components/modules/UserOnboardingModal/PlatformOnboarding/PageOne/PageOne.js

```diff
@@ -1,5 +1,14 @@
 import { Typography } from "@devlaunchers/components/components/atoms";
-import welcomePageOnbaordingLogo from "./../../../../../images/welcomePageOnboardingLogo.png";import { PageOneContainer } from "./StyledPageOne";
+import "/Users/rinabo/developments/dev-launchers-platform/apps/user-profile/src/images/logo-monogram.png" 
+import LogoMonogram from '../../../../../images/logo-monogram.png'
+import { motion } from "framer-motion";
```

Comment:
Hi @rinagobo , First of all I want to thank you for making the website more lively by adding animations!
just wanted to let you know that using motion is discouraged if you're only doing domAnimations so please read this https://www.framer.com/motion/guide-reduce-bundle-size/ and let me know what you think :rocket:

Reply:
Hi Mohammed, thank you for checking my code. I read through the document you attached. Is the easiest way to minimize the bundle size of my animation is putting <LazyMotion strict></LazyMotion> around my <motion.img /> and <motion.div /> ?

Reply:
The easiest way is to change the import statement from
`import {motion} from "framer-motion"` to `import {m} from "framer-motion"`

And<motion.div> to <m.div> etc

Then eventually wrap your<m.> Component with <LazyMotion features={domAnimation}>

Reply:
Thank you for your explanation! 

When do you write code like the one in the bottom of the document (SS attached)? I thought this could work too.

<img width="729" alt="Screenshot 2023-07-09 at 5 22 29 PM" src="https://github.com/dev-launchers/dev-launchers-platform/assets/105747673/741bb52e-b463-4681-9516-ae6f359d41e4">

Reply:
You can do that too and wrap your whole app with lazymotion with strict and features prop passed in, _app.jsx in the pages folder is where you should place lazymotion component around the {children}